archive/2022-01-27-DssSpell/Goerli-DssSpell.t.sol

```diff
@@ -142,7 +138,7 @@ contract DssSpellTest is GoerliDssSpellTestBase {
 
         ClipAbstract clipLINKA = ClipAbstract(addr.addr("MCD_CLIP_LINK_A"));
         assertEq(clipLINKA.wards(address(pauseProxy)), 1);
-        DenyProxyLike(address(esm)).denyProxy(address(clipLINKA));
+        ESMAbstract(address(esm)).denyProxy(address(clipLINKA));
```

Comment:
nit: I wouldn't introduce an interface that wasn't present in #42 yet (https://github.com/makerdao/dss-interfaces/pull/65).

Reply:
Yeah, since this is just a test I think the benefit of having someone in the future using the good version is worth it.
Not a strong opinion in any case.